css/style.css

```diff
@@ -0,0 +1,1194 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;1,300&display=swap");
```

Comment:
- [ ] Following the project requirements, the pages should use the same fonts from the original design. It would be nicer to use the `Cocogoose` font for the main title of the pages.